src/app/components/views/homepage/homepage.component.ts

```diff
@@ -61,6 +63,7 @@ export class HomepageComponent implements OnInit, OnDestroy {
       next: (loading: boolean) => {
         if(!loading && this.jobService.hasSuccessfullyCompleted()) {
           this.jobsResponse = this.jobService.getResults();
+          this.jobCount = this.jobsResponse.jobs.length;
```

Comment:
What if jobResponse isn't there?  Or what if it is and there or no jobs?

jobCount wouldn't be assigned as a hard stop error would occur. 

As in "cannot access jobs of undefined"

Reply:
added truthy checks before assigning jobCount. Decided jobCount should be set to something